addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -565,6 +585,9 @@ private String readReplyFromDevice(RequestMessage requestMessage) throws IOExcep
         private DeviceConnection serial1Connection;
         private DeviceConnection serial2Connection;
 
+        private SerialReader serialReader1;
```

Comment:
I assume these one and two also refer to the ports, please change the naming here also like that.